src/routes/tools/extreme-precipitation/_store.js

```diff
@@ -0,0 +1,125 @@
+import { writable, derived } from "svelte/store";
+import { merge } from "d3-array";
+import climvars from "~/helpers/climate-variables";
+import { makeCustomWritableStore } from "../_common/stores";
+import {
+  CLIMATE_INDICATORS,
+  DEFAULT_CLIMATE_VARIABLE,
+  DEFAULT_CLIMATE_INDICATOR,
+  DEFAULT_DURATION,
+  DEFAULT_RETURN_PERIOD,
+  DEFAULT_THRESHOLD_TYPE,
+  DEFAULT_POLYGON_AGGREGATE_FUNCTION,
+  WARNING_LOW_SAMPLE_SIZE,
+  WARNING_MISSING_CI,
```

Comment:
the imports for `WARNING_MISSING_CI` and `WARNING_LOW_SAMPLE_SIZE` are unused so could be removed.

Reply:
removed in https://github.com/berkeley-gif/caladapt-website-2021/pull/211/commits/261e217a662f28038051c7524d2aff378af60b4c